dotCMS/src/main/java/com/dotcms/languagevariable/business/LanguageVariableAPI.java

```diff
@@ -18,6 +18,8 @@
  *
  */
 public interface LanguageVariableAPI {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1214)